src/app/features/addresses/components/molecules/AddressMenu/AddressMenu.tsx

```diff
@@ -1,30 +1,39 @@
 import React from "react"
 import styled from "styled-components"
-import { themeSpacing } from "@datapunt/asc-ui"
+import { themeSpacing, breakpoint } from "@datapunt/asc-ui"
 import NavBlock from "app/features/addresses/components/atoms/NavBlock/NavBlock"
 import to from "app/features/shared/routing/to"
 import routesObject from "app/config/routes"
 
 type Props = {
   bagId: Components.Schemas.Address["bag_id"]
 }
-
 const Menu = styled.menu`
   width: 100%;
-  margin:  ${ themeSpacing(4) } 0;
+  margin: 0;
   padding: 0;
 `
 const Ul = styled.ul`
-  margin: 0;
+  margin: 0 -${ themeSpacing(1.5) };
   padding: 0;
   list-style: none;
   display: flex;
   flex-flow: row wrap;
   justify-content: space-between;
-
-  li {
-    margin-bottom: ${ themeSpacing(4) };
+`
+const Li = styled.li`
+  display: inline-block;
+  width: 100%;
+  margin-bottom: ${ themeSpacing(3) };
+  @media screen and ${ breakpoint("min-width", "tabletS") } {
```

Comment:
I thought we would use only 1 breakpoint ("laptop"). Or is this an exception?

Reply:
Yes, mobile has one column, tablet 2 and desktop 4